backend/medtagger/api/scans/service_rest.py

```diff
@@ -73,24 +73,56 @@ def post() -> Any:
         return business.create_scan_category(key, name, image_path), 201
 
 
+@scans_ns.route('/tasks')
```

Comment:
Maybe we should put it under a separate namespace?
Now it's URL path is `/api/v1/scans/tasks`. Shouln't it be `/api/v1/tasks`?